tourPlanProcess.php

```diff
@@ -0,0 +1,21 @@
+<?php
+require 'connection.php';
+
+$jsonText = $_POST["jsonText"];
```

Comment:
You must validate this `jsonText` parameter before assigning to a variable. It has some possibility that the parameter does not set to associated array or it can be the empty parameter.